### ISSUE TITLE
Content: Armstrong Number

### ISSUE THREAD

**Issue by HarshCasper** [TesseractCoding/NeoAlgo-Docs#25]
## Description 

Add the article for `Armstrong Number`. The basic layout for the article should be: 

- What is Armstrong Number?
- Explanation for Armstrong Number
- Algorithmic Steps 
- Code (Use C++/Java and Python; At least two languages should be present and pick up code from NeoAlgo with apt credits to author)
- Sample I/O 
- Time & Space Complexity 

This article should be between 500-700 words in length. You should use traditional Markdown to prepare the content and the use of external images is prohibited. You can however create your own images. Run a proper Grammarly Lint check before submitting the PR.

**Comment by deeptea22** [comment]
Hey I would like to work on this issue. Please assign this to me. Thank you :)

**Comment by HarshCasper** [comment]
Hi @deeptea22 

Thanks for taking this up! I hope you spend sometime in understanding the directory structure and where to make the PR. If you are confused, check it right here: https://github.com/TesseractCoding/NeoAlgo-Docs/tree/main/docs/mathematics

I have already created a file for you named `armstrong-number.md`. And I have added the same on the sidebar. So I guess you will have a fair understanding of how to contribute (which is much different from NeoAlgo)
